bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBatch.java

```diff
@@ -366,4 +375,12 @@ public void testBatchGets() throws Exception {
 
     table.close();
   }
+
+  public void testBatchDoesntHang() throws Exception {
+    Connection closedConnection = ConnectionFactory.createConnection(IntegrationTests.getConfiguration());
+    Table table = closedConnection.getTable(TABLE_NAME);
+    closedConnection.close();
+    thrown.expect(RetriesExhaustedWithDetailsException.class);
```

Comment:
I guess so.  The point here is that the exception is thrown and the put doesn't hang forever.